docs/src/main/asciidoc/_developer-guide/JPA_Reference_Guide.adoc

```diff
@@ -411,8 +406,7 @@ The Hibernate native API application will also need a MANIFEST.MF:
 Dependencies: org.infinispan,org.hibernate
 ....
 
-http://infinispan.org/docs/8.0.x/user_guide/user_guide.html#_using_infinispan_as_jpa_hibernate_second_level_cache_provider[Infinispan
-Hibernate/JPA second level cache provider documentation] contains
+http://infinispan.org/docs/9.2.x/user_guide/user_guide.html#jpa_hibernate_2l_cache contains
```

Comment:
The Infinispan doc referenced above shows "Hibernate documentation" linking to Hibernate ORM 5.2: https://docs.jboss.org/hibernate/orm/5.2/userguide/html_single/Hibernate_User_Guide.html#caching

Should the Infinispan doc be updated as well?

Reply:
I'll update to http://infinispan.org/docs/9.3.x/user_guide/user_guide.html#jpa_hibernate_2l_cache which seems to reference ORM 5.3